docs/dev/contribute.rst

```diff
@@ -87,15 +87,24 @@ projects, feel free to suggest the language on Transifex to start the process.
 
 .. _our projects on Transifex: https://www.transifex.com/projects/p/readthedocs/
 
-Triaging tickets
-----------------
+Triaging issues
+---------------
 
-Core team uses the following guidelines for issue triage on all of our projects.
-These guidelines describe the issue lifecycle, and what is required at each step
-of this lifecycle.
+.. rstcheck: ignore-roles="external+rtd:doc"
```

Comment:
Is this needed here? I saw it in the `.rstcheck.cfg` file as well.

Reply:
Sorry no, this is an artifact from a previous attempt :)